src/Microsoft.DotNet.Build.Tasks/CleanupVSTSAgent.cs

```diff
@@ -25,13 +26,14 @@ public class CleanupVSTSAgent : Microsoft.Build.Utilities.Task
         public int? SleepTimeInMilliseconds { get; set; }
 
         private static readonly int s_DefaultRetries = 5;
-        private static readonly int s_DefaultSleepTime = 10000;
+        private static readonly int s_DefaultSleepTime = 2000;
+        private static string[] processNamesToKill = new string[] { "git" };
```

Comment:
Can you make this...

`public ITaskItem[] ProcessNamesToKill { get; set;}`

... and then update https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/vstsagent/cleanupagent.proj

```
<ItemGroup>
  <ProcessNamesToKill Condition="'@(ProcessNamesToKill)' == ''" Include="git;msbuild;vbcscompiler" />
</ItemGroup>
```

It should also probably be an opt-in thing.

`public bool KillRunningProcesses { get; set; }`